app/controllers/api_controller/service_orders.rb

```diff
@@ -0,0 +1,15 @@
+class ApiController
+  module ServiceOrders
+    def clear_resource_service_orders(type, id, _data)
+      service_order = resource_search(id, type, collection_class(type))
+      # service_order.clear
+    rescue
```

Comment:
only rescue the service_order.clear, and use its message => e,  e.message. resource_search can raise its own Forbidden